doc/tutorials/mount.md

```diff
@@ -1,56 +1,146 @@
-Elektra provides a global key database where configuration of all
-applications can be integrated.
+# Mounting #
 
-There are different forms of integration.  In this tutorial we will
-discuss a light form of integration that can be achieved without
-elektrifying applications.
+Elektra provides a global key database where configuration of all applications can be integrated.
 
-The idea is to manipulate the configuration values directly in the
-configuration files via the key database.  Thus the applications will
-read the configuration file, changes in the key database will be picked
-up by applications.
+There are different forms of integration. In this tutorial we will discuss a light form of integration that can be achieved without [elektrifying](/doc/help/elektra-glossary.md) applications.
```

Comment:
I carried it over from the initial tutorial, which you wrote, i think.

Do I express what you meant with integration, if I rephrase this sentence to this?
> There are different forms of integrating Elektras configuration into applications.

If you mean somthing special with integration I could add it to the glossary and link it.